### ISSUE TITLE
glTF import crashes when importing models with cameras

### ISSUE THREAD

**Issue by Moguri** [godotengine/godot#10780]
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
OS: Arch Linux 64 bit
Git hash: 8337d272ba36044cc9e1a9273b01418a0120be9b

**Issue description:**
Importing the Duck sample from Khronos's glTF 2.0 samples (https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/Duck/glTF-Embedded/Duck.gltf) crashes Godot when importing. A backtrace (below) points to camera parsing. Editing the file to remove the camera works around the crash.

```
#0  0x00007f571eb978c0 in raise () from /usr/lib/libc.so.6
#1  0x00007f571eb98f72 in abort () from /usr/lib/libc.so.6
#2  0x00007f571ebd6be2 in __libc_message () from /usr/lib/libc.so.6
#3  0x00007f571ec63158 in __fortify_fail () from /usr/lib/libc.so.6
#4  0x00007f571ec63120 in __stack_chk_fail_local () from /usr/lib/libc.so.6
#5  0x0000555555de11de in EditorSceneImporterGLTF::_parse_cameras (this=<optimized out>, state=...) at editor/import/editor_scene_importer_gltf.cpp:1436
#6  0x0000555555de74ef in EditorSceneImporterGLTF::import_scene (this=0x55555afac980, p_path=..., p_flags=<optimized out>, p_bake_fps=15, r_missing_deps=<optimized out>, r_err=<optimized out>) at editor/import/editor_scene_importer_gltf.cpp:2085
#7  0x0000555555e00b16 in ResourceImporterScene::import (this=<optimized out>, p_source_file=..., p_save_path=..., p_options=..., r_platform_variants=<optimized out>, r_gen_files=<optimized out>) at editor/import/resource_importer_scene.cpp:1019
#8  0x0000555555bf7498 in EditorFileSystem::_reimport_file (this=<optimized out>, p_file=...) at editor/editor_file_system.cpp:1403
#9  0x0000555555bede48 in EditorFileSystem::reimport_files (this=0x55555afaccc0, p_files=...) at editor/editor_file_system.cpp:1504
#10 0x0000555555bece04 in EditorFileSystem::_update_scan_actions (this=<optimized out>) at editor/editor_file_system.cpp:393
#11 0x0000555555bf31cd in EditorFileSystem::_notification (this=0x55555afaccc0, p_what=<optimized out>) at editor/editor_file_system.cpp:1017
#12 0x0000555555bfce27 in EditorFileSystem::_notificationv (this=0x55555afaccc0, p_notification=17, p_reversed=<error reading variable: access outside bounds of object referenced via synthetic pointer>) at ./editor/editor_file_system.h:100
#13 0x0000555556d45801 in Object::notification (this=0x55555afaccc0, p_notification=17, p_reversed=<optimized out>) at core/object.cpp:849
#14 0x00005555562ea3a9 in SceneTree::_notify_group_pause (this=0x555559802f50, p_group=..., p_notification=17) at scene/main/scene_tree.cpp:908
#15 0x00005555562ea90b in SceneTree::idle (this=<optimized out>, p_time=<optimized out>) at scene/main/scene_tree.cpp:488
#16 0x00005555556fccf8 in Main::iteration () at main/main.cpp:1572
#17 0x00005555556e5062 in OS_X11::run (this=0x7fffffffe0b8) at platform/x11/os_x11.cpp:2109
#18 0x00005555556dd1d0 in main (argc=4, argv=0x7fffffffe5c8) at platform/x11/godot_x11.cpp:54
```

**Steps to reproduce:**
Attempt to import [this](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/Duck/glTF-Embedded/Duck.gltf) glTF file.

**Comment by Moguri** [comment]
This looks like a duplicate of #10176

**Comment by akien-mga** [comment]
Did this get fixed by ded74ded?

**Comment by reduz** [comment]
yes, should be fixed, closing
